config/crd/bases/osp-director.openstack.org_openstackbaremetalsets.yaml

```diff
@@ -147,8 +147,8 @@ spec:
                 secret data'
               type: string
             provisionServerName:
```

Comment:
Should the description say `ProvisionServerName` or `OpenStackProvisionServerName`?  Not sure whether the key name or the comment name is right.

Reply:
I would going to leave this just 'provisionServerName' for brevity.

Reply:
Ack